Seneca-CDOT/telescope
# Fixes #3615: Add author, title, publishDate fields to '/' route

## Issue This PR Addresses
Closes #3615 


## Type of Change




## Description
In this PR, an extra query param `expand` is added to the URL /posts in posts services. 
When `expand=1`, there will be extra data of author, title, and publishDate returned.
When expand does not equal 1, or expand does not present in the request, the request will return the data like before.


## Steps to test the PR
Load the posts page in the react-native app, and observe if the extra fields of data are returned by `/posts`. This requires the react-native app to add "expand=1" to its request query params of `/posts` as well.

Here's what does the request and data look like.
![image](https://user-images.githubusercontent.com/8223139/202312171-0f3c2eca-fe9c-4eac-b8a6-70d36bad6bd0.png)



## Checklist
